docs/src/guides/gatsby/headless/strapi.md

```diff
@@ -33,7 +33,7 @@ $ yarn add strapi-plugin-graphql
 
 {{< guides/gatsby/headless-gatsby name="Strapi" template="gatsby-strapi" >}}
 
-You can then modify [`gatsby-config.js`](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-config/) to read from the backend Strapi container through the `strapi` relationship defined above to configure the `apiURL` attribute for `gatsby-source-strapi`. Notice that the source plugin requires that you explicitly define the `contentTypes` you would like to retrieve from Strapi. At this point you have not built out the API, and the Content Types `article` and `category` are included to support the [post-install instructions](https://github.com/platformsh-templates/gatsby-strapi/tree/update#post-install) outlined in the template's README. Adjust these values to fit your current API if your are planning on migrating an existing Strapi repository. 
+You can then modify [`gatsby-config.js`](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-config/) to read from the backend Strapi container through the `strapi` relationship defined above to configure the `apiURL` attribute for `gatsby-source-strapi`. Notice that the source plugin requires that you explicitly define the `contentTypes` you would like to retrieve from Strapi. At this point you have not built out the API, and the Content Types `article` and `category` are included to support the [post-install instructions](https://github.com/platformsh-templates/gatsby-strapi/tree/update#user-content-post-install) outlined in the template's README. Adjust these values to fit your current API if your are planning on migrating an existing Strapi repository. 
```

Comment:
```suggestion
You can then modify [`gatsby-config.js`](https://www.gatsbyjs.com/docs/reference/config-files/gatsby-config/) to read from the backend Strapi container through the `strapi` relationship defined above to configure the `apiURL` attribute for `gatsby-source-strapi`.
Note that the source plugin requires that you explicitly define the `contentTypes` you would like to retrieve from Strapi. 
At this point, you have not built out the API, and the Content Types `article` and `category` are included to support the [post-install instructions](https://github.com/platformsh-templates/gatsby-strapi/tree/update#user-content-post-install) outlined in the template's README.
Adjust these values to fit your current API if your are planning on migrating an existing Strapi repository. 
```

Reply:
The wording is a bit heavy and would need some improvements, adding linebreaks for easier future editing and readability.